client/my-sites/domains/domain-management/transfer/transfer-out/icann-verification.jsx

```diff
@@ -13,52 +14,52 @@ import Button from 'components/button';
 import notices from 'notices';
 import support from 'lib/url/support';
 
-const IcannVerification = React.createClass( {
-	getInitialState() {
-		return {
-			submitting: false
-		};
-	},
+class IcannVerification extends React.Component {
+	state = {
+		submitting: false
+	};
 
-	handleClick() {
+	handleClick = () => {
 		this.setState( { submitting: true } );
 
 		resendIcannVerification( this.props.selectedDomainName, ( error ) => {
 			if ( error ) {
 				notices.error( error.message );
```

Comment:
Sure 👍 I was postponing fixing that one, because I misunderstood the scope of that issue originally (context: https://github.com/Automattic/wp-calypso/pull/15880#issuecomment-313766773). But it should be pretty straightforward if it's just "reduxifying" them.